PDF_UA/WCAG-21-Complete.xml

```diff
@@ -219,7 +219,7 @@
             </error>
             <references/>
         </rule>
-        <rule object="SAHn">
+        <rule object="SAHn" tags="minor,structure,heading">
```

Comment:
I think we should add "human" tag to all rules that use semantic score. I stop the review here as we need first to fix globally the following issues:
- add machine and human tags
- add a new severity level "cosmetic" for non-WCAG accessibility recommendations (merged cells, use of underline text, unwanted fonts, repeated characters)
- make sure we use  "syntax" only for rules related to PDF syntax (COS and PD levels, mainly machine rules)